test/metabase/test/data/sql.clj

```diff
@@ -240,21 +240,23 @@
         inline-comment (inline-column-comment-sql driver field-comment)]
     (str/join " " (filter some? [field-name field-type not-null unique inline-comment]))))
 
-(defn fielddefs->pk-field-name
-  "Find the pk field name in fieldefs"
+(defn fielddefs->pk-field-names
+  "Find the pks field name in fieldefs"
```

Comment:
This comment is the only place you pluralize "pk", and it's inconsistent with the function name:

```suggestion
  "Find the pk field names in fieldefs"
```